charts/ztka/templates/_helpers.tpl

```diff
@@ -126,12 +126,14 @@ Get DSN
 {{- define "ztka.dsn" -}}
   {{- if .Values.deploy.postgresql.enable -}}
 postgres://{{ .Values.postgresql.auth.username }}:{{ .Values.postgresql.auth.password }}@{{.Release.Name}}-postgresql.{{.Release.Namespace}}.svc.cluster.local:5432/{{ .Values.postgresql.auth.database }}?sslmode=disable
-  {{- else -}}
+  {{- else if empty .Values.deploy.postgresql.dsn -}}
```

Comment:
Ahh, I guess this would be a good addition. Just a few thoughts.

- We should probably document this option in `values.yaml` and mention that this is a alternative to specifying the individual values (probably also how the precedence works).
- This will fail as of now if the other values are also provided because of https://github.com/paralus/helm-charts/blob/1fe5934d76b605e77e0c20ceee1d7539223259e6/charts/ztka/templates/_helpers.tpl#L86

@akshay196 , anything you would like to add?

Reply:
Crated issue on Paralus: https://github.com/paralus/paralus/issues/45

Reply:
I guess we can maybe split up this up.
- PR#1 would just fix the password not being properly added
- PR#2 will add option to set DSN once we accept DSN in paralus/paralus

Reply:
Make sense.
@mcfearsome Can you please make changes suggested by @meain? Then we are good to go.